vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -2,13 +2,13 @@
 sidebarDepth: 2
 ---
 
-# Entando Bundle Transition to 7.1 
+# Entando Bundle Transition to 7.1  Entando Bundle Evolution
 
-Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to Docker-based OCI bundles, and this page clarifies the differences between the two, along with the improvements that were made with the new release.
+Entando 7.1 implements a more modular approach to building apps with the **ent bundle** CLI orchestrating the management of project bundles. The Entando bundle was restructured, its processes decoupled by component type, and the publishing transport method was streamlined. With these changes, Entando Bundles went from git-based to Docker-based OCI bundles. Though both systems are supported on Entando 7.1, this page clarifies the differences between the two, along with the improvements that were made with the new release.
```

Comment:
Entando bundle -> Entando Bundle (2nd sentence)
Docker-based -> docker-based (4th line)
system is not the right word here. implementations is an option

Reply:
I'll change it to methods, shorter than implementation

Reply:
it's actually not a method, though... implementations is really the best word here. unless you want to use "bundle structures"

same comment as before applies to "went from"; don't know if you saw it. we need to make it clear that v1 and v5 bundles both exist, and that v1 bundles have not been converted over to v5. 7.1 introduces a new bundle format, but this does not impact existing bundles from 7.0 backwards. the conversion to turn v1 into v5 has to be done by the user